lib/pages/home/home_page.dart

```diff
@@ -1,18 +1,55 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:get/get.dart';
+import 'package:intl/intl.dart';
 import 'package:survey/pages/home/home_controller.dart';
 
-import '../../flavors.dart';
-
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: GetBuilder(
         init: HomeController(),
-        builder: (controller) => Center(
-          child: Text(
-            'Home Page ${F.title}',
+        builder: (controller) => Container(
+          height: double.infinity,
+          decoration: BoxDecoration(
+            color: Colors.black.withOpacity(0.4),
+          ),
+          child: SafeArea(
+            child: Padding(
+              padding: const EdgeInsets.only(left: 24, right: 24),
+              child: Wrap(
+                children: [
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      // TODO: Bind real data (date & user profile) later in [Integrate].
+                      new Column(
```

Comment:
`new` keyword is optional in [Dart 2 ](https://dart.dev/guides/language/language-tour#using-constructors) 😄